lib/init.coffee

```diff
@@ -16,7 +16,7 @@ module.exports =
       @subscriptions.add atom.config.observe 'linter-htmlhint.executablePath',
         (executablePath) =>
           @executablePath = executablePath
-      @scopes =  ['text.html.angular', 'text.html.basic', 'text.html.erb', 'text.html.gohtml', 'text.html.jsp', 'text.html.mustache', 'text.html.handlebars', 'text.html.php', 'text.html.ruby']
+      @scopes =  ['text.html.angular', 'text.html.basic', 'text.html.erb', 'text.html.gohtml', 'text.html.jsp', 'text.html.mustache', 'text.html.php', 'text.html.ruby']
```

Comment:
Sorry I missed that as I did copy paste from the corresponding init.coffee
that is installed on my Atom Editor.
It's strange ,though , as I did a clean install yesterday of the Atom and
its .atom folder.

So please add 'text.html.handlebars' back. :)
Thanks

On Thu, Oct 22, 2015 at 1:14 AM, Landon Abney <notifications@github.com>
wrote:

> In lib/init.coffee
> <https://github.com/AtomLinter/linter-htmlhint/pull/39#discussion_r42690142>
> :
>
> > @@ -16,7 +16,7 @@ module.exports =
> >        @subscriptions.add atom.config.observe 'linter-htmlhint.executablePath',
> >          (executablePath) =>
> >            @executablePath = executablePath
> > -      @scopes =  ['text.html.angular', 'text.html.basic', 'text.html.erb', 'text.html.gohtml', 'text.html.jsp', 'text.html.mustache', 'text.html.handlebars', 'text.html.php', 'text.html.ruby']
> > +      @scopes =  ['text.html.angular', 'text.html.basic', 'text.html.erb', 'text.html.gohtml', 'text.html.jsp', 'text.html.mustache', 'text.html.php', 'text.html.ruby']
>
> Did this not support text.html.handlebars properly in the first place, or
> why did you remove that?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/AtomLinter/linter-htmlhint/pull/39/files#r42690142>.
>